python_modules/dagster-airflow/dagster_airflow/dagster_plugin.py

```diff
@@ -0,0 +1,268 @@
+# This modified DockerOperator incorporates https://github.com/apache/airflow/pull/4315/files
```

Comment:
So post this PR in the main airflow repo we can just reuse the DockerOperator?

Reply:
Can we have a separate class called Post4315DockerOperator so we can tease apart what is dagster-specific versus what is PR 4315-specific. I think this should structured so that when 4135 is merged we can remove the copy-paste job cleanly